backport/core:  No kernel_addons headers in public infiniband header files

Eliminate includes of header files located only in the
kernel_addons/backport directories in publicly-visible
ofa header files (e.g., include/rdma/*.h), so as not to force
kernel applications to directly access the kernel_addons/backport
directories themselves.

Signed-off-by: Jack Morgenstein <jackm@dev.mellanox.co.il>

---
 include/rdma/ib_verbs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: ofed_kernel/include/rdma/ib_verbs.h
===================================================================
--- ofed_kernel.orig/include/rdma/ib_verbs.h
+++ ofed_kernel/include/rdma/ib_verbs.h
@@ -46,7 +46,7 @@
 #include <linux/kref.h>
 #include <linux/list.h>
 #include <linux/rwsem.h>
-#include <linux/scatterlist.h>
+#include <asm/scatterlist.h>
 
 #include <asm/atomic.h>
 #include <asm/uaccess.h>
